src/software/SfM/main_ConvertAnimatedCamera.cpp

```diff
@@ -0,0 +1,62 @@
+#include "openMVG/sfm/sfm.hpp"
+#include <openMVG/sfm/AlembicExporter.hpp>
+#include "third_party/cmdLine/cmdLine.h"
+#include "third_party/stlplus3/filesystemSimplified/file_system.hpp"
+#include <string>
+#include <vector>
+
+using namespace openMVG;
+using namespace openMVG::sfm;
+
+int main(int argc, char **argv)
+{
+  CmdLine cmd;
+
+  std::string
+    sSfM_Data_Filename_In,
+    sSfM_Data_Filename_Out;
+
+  cmd.add(make_option('i', sSfM_Data_Filename_In, "input_file"));
+  cmd.add(make_option('o', sSfM_Data_Filename_Out, "output_file"));
+
+  try {
+      if (argc == 1) throw std::string("Invalid command line parameter.");
+      cmd.process(argc, argv);
+  } catch(const std::string& s) {
+      std::cerr << "Usage: " << argv[0] << std::endl;
+      std::cerr  << "[-i|--input_file] path to the input ground truth folder" << std::endl;
+      std::cerr << "[-o|--output_file] path to the output Alembic file" << std::endl;
+
+      std::cerr << s << std::endl;
+      return EXIT_FAILURE;
+  }
+
+  if (sSfM_Data_Filename_In.empty() || sSfM_Data_Filename_Out.empty())
+  {
+    std::cerr << "Invalid input or output filename." << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Load input SfM_Data scene
+  SfM_Data sfm_data;
+  if (!Load(sfm_data, sSfM_Data_Filename_In, ESfM_Data(ALL)))
```

Comment:
this will only work if we patch `readGt()` so that it does not use UID. Not really "proper" to have a program that only works if you change the code and recompile.

Maybe a cleaner solution, if we want to keep the binary, is to add a parameter `bool useUID` to `readGt()` which by default is set to `true`. This allows to use or not UID in `readGt()`. Then here, we can call directly `readGt()` with the parameter set to false, instead of using `Load()`, as we only take a folder as input `--input_file` (and maybe it's worth to add a check in this code to see if it is a folder and it exists).
@caymard @fabiencastan